core/bones/__init__.py

```diff
@@ -8,6 +10,7 @@
 from .file import FileBone
 from .json import JsonBone
 from .key import KeyBone
+from ._lambda import LambdaBone
```

Comment:
Should we better name this `lambda_`? The preceding `_` always looks a little bit private...